server/functional-test/src/main/java/org/zanata/page/administration/ManageUserAccountPage.java

```diff
@@ -52,6 +53,17 @@ public ManageUserAccountPage(WebDriver driver) {
         roleMap.put("user", "4");
     }
 
+    public ManageUserAccountPage enterFullName(String fullName) {
+        log.info("Enter name {}", fullName);
+        enterText(readyElement(fullNameField), fullName);
+        return new ManageUserAccountPage(getDriver());
+    }
+
+    public String getCurrentName() {
+        log.info("Query user's name");
+        return readyElement(fullNameField).getAttribute("value");
+    }
+
     public ManageUserAccountPage enterPassword(String password) {
         log.info("Enter password {}", password);
```

Comment:
Seems like the above log messages should be debug, not info. And this (old) log message shouldn't include a password.